netket/utils/group/cubic.py

```diff
@@ -85,3 +85,14 @@ def Oh() -> PointGroup:
 
 
 octahedral = cubic = Oh
+
+
+def Fd3m() -> PointGroup:
+    """Nonsymmorphic "point group" of the diamond and pyrochlore lattices with a
+    cubic unit cell of side length 1."""
+    return (_inv_group().change_origin([1 / 8, 1 / 8, 1 / 8]) @ Td()).replace(
```

Comment:
Out of curiosity why not do this with Tetrahedral group + glide?

Reply:
I think this is physically more transparent. Td contains a number of reflections, so you'd have to look for a reflection that is not in it, and work out what glide it corresponds to (if any). OTOH it's quite clear that Td on diamond sites and inversion across pyrochlore sites generate the space group.

Reply:
Ahh i see. Now that I think about it the tetrahedron point group doesn't commute with the origin shift so doing it a different way would be a complete mess.